content/projects/memory-game/part-1/_index.md

```diff
@@ -38,4 +38,10 @@ Build your game in a TDD manner. If you don't include tests in your final submis
 Tests are very important. Please follow the following best practices:
 
 - tdd!!!
-- if you are using plain ol js then your tests need to make use of mocks and spies to check that the frontend is being updated as it should be
\ No newline at end of file
+- if you are using plain ol js then your tests need to make use of mocks and spies to check that the frontend is being updated as it should be
+
+## Instructions for the reviewer
+
+- Check that once cards are matched, they cannot be flipped over again.
+- Check the good naming conventions are followed for variables, files, functions, classes and so on.
```

Comment:
Review instructions should be project-specific, clean code guidelines should always be followed, therefore they don't need to be mentioned